ST_mortality.c

```diff
@@ -68,70 +71,74 @@ void _kill_maxage(void);
 
 /************ File-Level Variable Declarations *************/
 /***********************************************************/
+/**
+ * \brief TRUE if a function in ST_mortality.c killed an individual.
+ * \ingroup MORTALITY_PRIVATE
+ */
 Bool _SomeKillage;
-/* flag: some plant was reduced and PR is affected. */
-/* 7/5/01  - currently flag is set but unused. */
+
 extern
   pcg32_random_t mortality_rng; //declared in ST_main.c
 
-
-/***********************************************************/
-/***********************************************************/
+/**
+ * \brief Performs mortality that occurs during the growing season.
+ * 
+ *  This function reduces "plant-space" according to
+ *  resource availability, age, growth rate, and disturbance
+ *  conditions.  Also, succulents are "mortified" if it's a wet
+ *  year.
+ * 
+ *  This function is not to be confused with mort_EndOfYear().
+ *
+ *  An important consideration is the fact that the age of new
+ *  plants starts at 1.  Consider that for it to establish, it
+ *  has already survived the massive mortality of year 0; this
+ *  rate of year 0 mortality, among other factors of survival,
+ *  is captured by the probability of establishment (see
+ *  rgroup_Establish()).  However, for consistency with other
+ *  arrays, the age-related arrays are indexed from 0.  Thus,
+ *  age is base1 but the arrays are base0.
+ * 
+ *  The outline for this function is:
+ *    1) Death by limited resources.
+ *    2) Age independent mortality.
+ *    3) Slow growth mortality.
+ *    4) Succulent wet season mortality.
+ *    5) Death by pats, mounds or burrows.
```

Comment:
5) Mortality due to fecal pats, ant mounds or animal burrows.

Reply:
Addressed by commit 984bd13.